appveyor.yml

```diff
@@ -66,11 +66,10 @@ test_script:
 
       if ($testsExitCode -ne 0) {exit $testsExitCode}
 
-  - ps: |
-      & coverage erase
+  - coverage erase
+  - del /Q coverage.xml
 
-      $clcachePy = "$((Get-Location).Path)\clcache.py"
-      $env:CLCACHE_CMD = "coverage run -a --branch --include=$clcachePy $clcachePy";
+  - ps: |
```

Comment:
You can run coverage perfectly recursively. The core is: either rely on the pytest-cov magic and hope it does the job well or configure everything manually and don't use pytest at all.